build/Signing/SignToolConfig.json

```diff
@@ -4,7 +4,20 @@
             "certificate": "Microsoft402",
             "strongName": "MsSharedLib72",
             "values": [
-                "Microsoft.NET.Build.Tasks.dll"
+                "Raw/Microsoft.NET.Build.Tasks.dll",
```

Comment:
>Microsoft.NET.Build.Tasks.dll [](start = 21, length = 29)

So did the location of this DLL move, or is it in both places? Did you check that it is still packaged properly etc.